### ISSUE TITLE
Update hackage

### ISSUE THREAD

**Issue by AlexeyRaga** [haskell-distributed/distributed-process-platform#98]
Hi Guys,

Is there a plan to update Hackage anytime soon?
Fixed ```fingertree``` version dependency and fixed issue with `Setup.hs` -> `Setup.lhs` would be of a great help.

**Comment by hyperthunk** [comment]
Yes sorry, I've been away for almost a month. We need to divvy up to some more maintainers so this doesn't keep happening. But I'll get Hackage updated this week.
